vita3k/modules/SceVideodec/SceVideodecUser.cpp

```diff
@@ -190,6 +190,7 @@ EXPORT(int, sceAvcdecDecode, SceAvcdecCtrl *decoder, const SceAvcdecAu *au, SceA
     const auto send = decoder_info->send(reinterpret_cast<uint8_t *>(au->es.pBuf.get(emuenv.mem)), au->es.size);
     if (send && decoder_info->receive(output)) {
         decoder_info->get_res(pPicture->frame.frameWidth, pPicture->frame.frameHeight);
```

Comment:
Can you remove this line? The frameWidth is provided by the user and should not be updated by this function call.